source/indexes/unique-index.txt

```diff
@@ -0,0 +1,59 @@
+.. _pymongo-unique-index:
+
+==============
+Unique Indexes
+==============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: index, query, optimization, efficiency
+
+Overview
+--------
+
+Unique indexes ensure that the indexed fields do not store duplicate values. By
+default, MongoDB creates a unique index on the ``_id`` field during the creation
+of a collection. To create a unique index, specify the field or combination of
+fields that you want to prevent duplication on and set the ``unique`` option to
+``True``.
+
+Sample Data
+~~~~~~~~~~~
+
+The examples in this guide use the ``sample_mflix.movies`` collection
+from the :atlas:`Atlas sample datasets </sample-data>`. To learn how to create a
+free MongoDB Atlas cluster and load the sample datasets, see the
+:ref:`<pymongo-get-started>`.
+
+Create a Unique Index
+---------------------
+
+The following example creates a unique, descending index on the ``theaterId`` field:
```

Comment:
The next section says "ascending wildcard index", with no comma. I'd probably use the same pattern here:

```suggestion
The following example creates a descending unique index on the ``theaterId`` field:
```